aws-android-sdk-iot/src/main/java/com/amazonaws/mobileconnectors/iot/AWSIotMqttManager.java

```diff
@@ -827,6 +843,81 @@ private void connect(KeyStore keyStore, int portNumber, final AWSIotMqttClientSt
         }
     }
 
+    /**
+     * Initializes the MQTT session and connects to the specified MQTT server
+     * using certificate and private key in keystore on the specified port.
+     *
+     * @param keyStore A keystore containing an keystore with a certificate and
+     *            private key. Use IotKeystoreHelper to get keystore.
+     * @param portNumber the client port, either 8883 or 443
+     * @param statusCallback When new MQTT session status is received the
+     *            function of callback will be called with new connection
+     *            status.
+     */
+    private void connect(KeyStore keyStore, int portNumber, final MqttConnectOptions connectOptions, final AWSIotMqttClientStatusCallback statusCallback) {
```

Comment:
This method is copy and paste of another overloaded method in this file. Can you please eliminate the redundancy?